tests/models/blip/test_modeling_blip.py

```diff
@@ -1381,6 +1381,17 @@ def test_inference_image_captioning_fp16(self):
             [30522, 1037, 3861, 1997, 1037, 2450, 1998, 2014, 3899, 2006, 1996, 3509, 102],
         )
 
+    def test_inference_interpolate_pos_encoding(self):
+        model = BlipForConditionalGeneration.from_pretrained("Salesforce/blip-image-captioning-base").to(torch_device)
+        processor = BlipProcessor.from_pretrained("Salesforce/blip-image-captioning-base")
+        processor.image_processor.size = {"height": 500, "width": 500}
+
+        image = prepare_img()
+        inputs = processor(images=image, return_tensors="pt").to(torch_device)
+
+        predictions = model.generate(**inputs, interpolate_pos_encoding=True)
+        self.assertEqual(predictions[0].tolist(), [30522, 1037, 2450, 3564, 2006, 1996, 3509, 2007, 1037, 3899, 102])
```

Comment:
Have you checked the generated ids here to make sure they make sense for the picture i.e.:

`processor.batch_decode([30522, 1037, 2450, 3564, 2006, 1996, 3509, 2007, 1037, 3899, 102])`